### ISSUE TITLE
Server implementation

### ISSUE THREAD

**Issue by Tuatan** [SolalPirelli/ThriftSharp#5]
Any plans to add a HTTP listener?

**Comment by SolalPirelli** [comment]
No, since I don't need one, but I'd gladly accept a PR. (Sorry, not much time to write one up / test it for now)
